e2e_tests/conftest.py

```diff
@@ -22,21 +27,49 @@ def pytest_sessionstart(session: pytest.Session) -> None:
 
     server = environ.get("E2E_ENVIRONMENT")
     api_key = environ.get("E2E_API_KEY")
+    team_slug = environ.get("E2E_TEAM")
 
     if server is None:
         raise E2EEnvironmentVariableNotSet("E2E_ENVIRONMENT")
 
     if api_key is None:
         raise E2EEnvironmentVariableNotSet("E2E_API_KEY")
 
+    if team_slug is None:
+        raise E2EEnvironmentVariableNotSet("E2E_TEAM")
+
     if not isinstance(session.config.cache, pytest.Cache):
         raise TypeError("Pytest caching is not enabled, but E2E tests require it")
 
     session.config.cache.set("server", server)
     session.config.cache.set("api_key", api_key)
+    session.config.cache.set("team_slug", team_slug)
+
+    global datasets
```

Comment:
Normally a very bad practice, this is one of the few exceptions, as this function can only run once in scope.  It's also the only way to solve this issue without storing the item in a custom format, and this is not pickleable.